navigation/ca_cartographer/launch/rviz_localization.launch

```diff
@@ -0,0 +1,4 @@
+<launch>
+  <node name="rviz" pkg="rviz" type="rviz" required="true"
```

Comment:
There is already an RViz launch file [here](https://github.com/RoboticaUtnFrba/create_autonomy/blob/kinetic-devel/ca_tools/launch/rviz.launch). Call that instead.